crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -118,7 +118,7 @@ impl SlidingSyncList {
             return Err(Error::CannotModifyRanges(self.name().to_owned()));
         }
 
-        self.inner.set_ranges::<UInt>(&[]);
+        self.inner.set_ranges::<u32>(&[]);
```

Comment:
(Now I'm using u32, so this is actually the right form heh :slightly_smiling_face:)

Not sure about the type alias; this is really the one place where it would be useful. Maybe this weird call format suggests we could have `set_empty_ranges` instead, so the caller doesn't have to worry about the underlying type?